Monika After Story/game/event-handler.rpy

```diff
@@ -2040,9 +2039,10 @@ init 1 python in evhand:
             ev - event to random
             rebuild_ev - True if we wish to notify idle to rebuild events
         """
-        ev.random = True
-        if kwargs.get("rebuild_ev", False):
-            store.mas_idle_mailbox.send_rebuild_msg()
+        if store.mas_bookmarks_derand.shouldRandom(ev.eventlabel):
```

Comment:
This is a pretty major core change. Is the issue that events are being re-random'd even though they shouldn't be random? What's causing them to have conditionals/actions that make them re-random?